lib/src/main/java/space/taran/arklib/Lib.kt

```diff
@@ -46,6 +71,14 @@ private external fun getLinkHashNative(url: String): String
 private external fun fetchLinkDataNative(url: String): LinkData?
 private external fun pdfPreviewGenerateNative(path: String, quality: String): Bitmap
 
+/*
+ *  Just to allow for preliminary testing with Navigator
+ *  A better way may be proposed to get application instance
+ *  Navigator should call this function in [Application.onCreate].
+ */
+fun Application.initArkLib() {
```

Comment:
Am I right, this function must be called from each app using `arklib-android`?

Reply:
Yes